thorlcr/msort/tsorts.cpp

```diff
@@ -798,12 +798,21 @@ class CThorSorter : public CSimpleInterface, implements IThorSorter, implements
     }
     ~CThorSorter()
```

Comment:
Should I add noexcept ?

Reply:
I'm not sure.  What do we want to do about unexpected exceptions?
Adding noexcept would make it a bit more resilient, but probably not much.
The only exceptions which are likely to be thrown are those derived from std::exception.  std::bad_alloc is a possibility, and I wonder if we should call std::set_new_handler to change the behaviour in that situation.  (I have been debating a change in eclcc since my regression suite keeps hitting it).

Reply:
I'll leave as is.